build_tools/github_actions/test_executable_scripts/test_miopen.py

```diff
@@ -182,12 +182,14 @@
 # If smoke tests are enabled, we run smoke tests only.
 # Otherwise, we run the normal test suite
 test_type = os.getenv("TEST_TYPE", "full")
-if test_type == "smoke":
-    test_filter = "--gtest_filter=" + ":".join(smoke_filter)
-else:
-    test_filter = (
-        "--gtest_filter=" + ":".join(positive_filter) + "-" + ":".join(negative_filter)
-    )
+test_filter = []
+if test_type != "full":
+    if test_type == "smoke":
+        test_filter = "--gtest_filter=" + ":".join(smoke_filter)
+    else:
+        test_filter = (
+            "--gtest_filter=" + ":".join(positive_filter) + "-" + ":".join(negative_filter)
+        )
```

Comment:
This line needs to be formatted: https://github.com/ROCm/TheRock/actions/runs/19528390272/job/55905697144?pr=2222#step:4:158

Please run pre-commit: https://github.com/ROCm/TheRock/blob/main/CONTRIBUTING.md#pre-commit-checks